tests/e2e/tests/pilot/egressgateway_test.go

```diff
@@ -69,6 +69,47 @@ func TestRouteHTTPViaEgressGateway(t *testing.T) {
 	}
 }
 
+func TestIstioMutualGatewayTLSMode(t *testing.T) {
+
+	//log.Errorf("AuthEnabled: %t, SdsEnabled: %t", tc.Kube.AuthEnabled, tc.Kube.AuthSdsEnabled)
```

Comment:
I think all integration tests should use new test framework. If there is a strong reason doing this and no objection from networking team, then I am fine. Please remove the commented code, and add a brief comment above TestIstioMutualGatewayTLSMode.